fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -82,17 +82,11 @@ public StarOSAgent() {
     }
 
     public boolean init(StarManagerServer server) {
-        if (Config.integrate_starmgr) {
-            if (!Config.use_staros) {
-                LOG.error("integrate_starmgr is true but use_staros is false!");
-                return false;
-            }
-            // check if Config.starmanager_address == FE address
-            String[] starMgrAddr = Config.starmgr_address.split(":");
-            if (!starMgrAddr[0].equals("127.0.0.1")) {
-                LOG.error("Config.starmgr_address not equal 127.0.0.1, it is {}", starMgrAddr[0]);
-                return false;
-            }
+        // check if Config.starmanager_address == FE address
```

Comment:
shall check current mode is sharednothing mode?

Reply:
No need.
![image](https://user-images.githubusercontent.com/10295131/220314307-2518c33e-3b48-4b22-b965-d2e0d4ab3920.png)
 Already checked before call init() of StarOSAgent.java